src/Microsoft.Extensions.Configuration/ChainedConfigurationProvider.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public class ChainedConfigurationProvider : ConfigurationProvider
+    {
+        private readonly IConfiguration _configuration;
+
+        public ChainedConfigurationProvider(IConfiguration configuration)
+        {
+            if (configuration == null)
+            {
+                throw new ArgumentNullException(nameof(configuration));
+            }
+            _configuration = configuration;
+        }
+
+        public override void Load()
```

Comment:
BTW, what should happen if the chained configuration is reloaded? Is it ok for the IConfigurations to return a mix of fresh and stale data from different providers?

Should have a way to propagate the reload like we do for file sources?